.ruby-version

```diff
@@ -1 +1 @@
-2.7.2
```

Comment:
Please stick with Ruby 2.7.2, as this is the default available installed version on your Netlify builds.

<https://github.com/netlify/build-image/blob/focal/included_software.md#languages>

A different version does work, but extends build time for each commit we make (which is what we don't want).

Reply:
👍